hackathon_site/event/views.py

```diff
@@ -226,3 +227,34 @@ class TeamListView(mixins.ListModelMixin, generics.GenericAPIView):
 
     def get(self, request, *args, **kwargs):
         return self.list(request, *args, **kwargs)
+
+
+class ProfileDetailView(mixins.RetrieveModelMixin, generics.GenericAPIView):
+
+    queryset = Profile.objects.all()
+    serializer_class = ProfileModifySerializer
```

Comment:
This only gets used if you use `mixins.UpdateModelMixin` (which will also run validators on the data). Right now there isn't any actual data validation going on in the plain `patch` method

Reply:
I'll change RetrieveModelMixin to UpdateModelMixin then